exchanges/bitmex/bitmex_wrapper.go

```diff
@@ -792,18 +792,24 @@ func (b *Bitmex) GetOrderHistory(ctx context.Context, req *order.GetOrdersReques
 		if orderType == "" {
 			orderType = order.UnknownType
 		}
+		pair := currency.NewPairWithDelimiter(resp[i].Symbol, resp[i].SettlCurrency, format.Delimiter)
 
 		orderDetail := order.Detail{
-			Price:    resp[i].Price,
-			Amount:   float64(resp[i].OrderQty),
-			Exchange: b.Name,
-			ID:       resp[i].OrderID,
-			Side:     orderSide,
-			Type:     orderType,
-			Status:   order.Status(resp[i].OrdStatus),
-			Pair: currency.NewPairWithDelimiter(resp[i].Symbol,
-				resp[i].SettlCurrency,
-				format.Delimiter),
+			Price:                resp[i].Price,
+			AverageExecutedPrice: resp[i].AvgPx,
```

Comment:
Reference: https://bitmex.freshdesk.com/en/support/solutions/articles/13000080520-execution

e.g. AvgPx = Average filled price of the order associated with this execution